ranking/get_common_data.go

```diff
@@ -0,0 +1,57 @@
+package ranking
+
+import (
+	"github.com/PretendoNetwork/nex-go"
+	ranking "github.com/PretendoNetwork/nex-protocols-go/ranking"
+)
+
+func getCommonData(err error, client *nex.Client, callID uint32, uniqueID uint64) uint32 {
+	if commonRankingProtocol.getCommonDataHandler == nil {
+		logger.Warning("Ranking::GetCommonData missing GetCommonDataHandler!")
+		return nex.Errors.Core.NotImplemented
+	}
+
+	server := client.Server()
+
+	if err != nil {
+		logger.Error(err.Error())
+		return nex.Errors.Ranking.InvalidArgument
+	}
+
+	commonData, err := commonRankingProtocol.getCommonDataHandler(uniqueID)
+
+	if err != nil {
+		return nex.Errors.Ranking.NotFound
+	}
```

Comment:
The empty space can be removed

```suggestion
	commonData, err := commonRankingProtocol.getCommonDataHandler(uniqueID)
	if err != nil {
		return nex.Errors.Ranking.NotFound
	}
```